'

```diff
@@ -0,0 +1,135 @@
+use std::collections::HashMap;
```

Comment:
Was this file added erroneously?

Reply:
We use the HashMap to collect a map of entry points to iterate over them in order to then run them. Line: 106